ksqldb-common/pom.xml

```diff
@@ -157,6 +157,11 @@
                 <filtering>true</filtering>
             </resource>
 
+            <resource>
```

Comment:
did you need to add this to ensure the properties file is included in the jar?

Reply:
without this it wouldn't resolve the `${project.version}` when I built the module